modules/ipi-install-network-requirements.adoc

```diff
@@ -13,20 +13,13 @@ Installer-provisioned installation of {product-title} involves several network r
 
 - `provisioning`: The `provisioning` network is an optional non-routable network used for provisioning the underlying operating system on each node that is a part of the {product-title} cluster. The network interface for the `provisioning` network on each cluster node must have the BIOS or UEFI configured to PXE boot.
 +
-In {product-title} 4.3, when deploying using the `provisioning` network, the first NIC on each node, such as `eth0` or `eno1`, must interface with the `provisioning` network.
-+
-In {product-title} 4.4 and later releases, you can specify the provisioning network NIC with the `provisioningNetworkInterface` configuration setting.
+In {product-title} versions 4.6 through 4.8, the `provisioningNetworkInterface` configuration setting specifies the `provisioning` network NIC. In {product-title} 4.9 and later releases, you can specify the `provisioning` network NIC with the `bootMACAddress` configuration setting.
 
 - `baremetal`: The `baremetal` network is a routable network.
 +
 In {product-title} 4.3, when deploying using the `provisioning` network, the second NIC on each node, such as `eth1` or `eno2`, must interface with the `baremetal` network.
 +
-In {product-title} 4.4 and later releases, you can use any NIC order to interface with the `baremetal` network, provided it is the same NIC order across worker and control plane nodes and not the NIC specified in the `provisioningNetworkInterface` configuration setting for the `provisioning` network.
-
-[NOTE]
-====
-Use a compatible approach such that cluster nodes use the same NIC ordering on all cluster nodes. NICs must have heterogeneous hardware with the same NIC naming convention such as `eth0` or `eno1`.
-====
+In {product-title} 4.4 and later releases, you can use any NIC to interface with the `baremetal` network, provided it is the same NIC order across worker and control plane nodes when using the `provisioningNetworkInterface` configuration setting. In {product-title} 4.9 and later releases, you may use any NIC provided it is not the NIC associated to the `bootMACAddress` configuration setting for the `provisioning` network when using the `bootMACAddress` configuration setting instead of the `provisioningNetworkInterface` setting.
```

Comment:
```suggestion
In {product-title} 4.4 and later releases, you can use any NIC to interface with the `baremetal` network, provided it is the same NIC order across worker and control plane nodes when using the `provisioningNetworkInterface` configuration setting. In {product-title} 4.9 and later releases, you can use any NIC, provided it is not the NIC associated to the `bootMACAddress` configuration setting for the `provisioning` network when using the `bootMACAddress` configuration setting instead of the `provisioningNetworkInterface` setting.
```